website/content/docs/operators/deployment/schedulers/yarn.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Apache Hadoop YARN Cluster (Experimental)
+---
+
+In addition to out-of-the-box scheduler for [Aurora](../aurora), Heron can also be deployed on a
+YARN cluster with the YARN Scheduler. YARN
+scheduler is implemented using Apache REEF framework.
+
+**Key features** of YARN scheduler:
+
+* **Heterogeneous container allocation:** YARN scheduler will request heterogeneous containers from
```

Comment:
In general, the use of articles needs to be enforced. It needs to be *the* YARN scheduler, *a* YARN scheduler, etc. Please make the appropriate changes throughout.